README.md

```diff
@@ -35,7 +42,7 @@ pyenv install 3.9.x
 
 Clone the repository
 ```
-git clone git@github.com:HewlettPackard/py-spiffe.git
+Follow through the steps in [Contributing](#Contributing) section.
```

Comment:
One detail, the text is inside a code block so the link style doent's work. It looks like this:

![image](https://user-images.githubusercontent.com/25386561/104469080-efb4f000-5596-11eb-9229-4487d0c9ce3b.png)

On a different note, referring to the whole contributing section looks to me like "exaggerated". there are other steps there that are not required for the purpose of just _get the repository cloned_ as the header suggest.

What about something like (just text, no code block style):

Clone the repository
Follow steps 1 and 2 from the [Contributing](#Contributing) section.

Reply:
Personally, I don't see the need to spell out the steps too explicitly - it's no different than any other github repository.

Reply:
While I do agree with Scott, I need to mention that the PR was originally created to add step two:
`2- Clone your forked repository (git clone git@github.com:<your_github_account>/py-spiffe.git)`

That is also a common path when working with Github. So if we add that for less experienced persons I think adding the correct steps here will make thing less confusing for them. That was my point here.

Said that, I'm Ok with the text as it is too.

Reply:
Done!